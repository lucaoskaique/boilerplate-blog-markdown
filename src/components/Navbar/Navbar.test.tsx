import { render, screen } from "@testing-library/react"
import { describe, expect, it } from "vitest"

import Navbar from "."

describe("<Navbar />", () => {
  it("should render the heading", () => {
    render(<Navbar />)

    expect(screen.getByRole("heading", { name: /Navbar/i })).toBeDefined()

    expect(document.body).toMatchSnapshot()
  })
})
