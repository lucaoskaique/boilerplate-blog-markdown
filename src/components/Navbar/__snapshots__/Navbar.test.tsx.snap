// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Navbar /> > should render the heading 1`] = `
<body>
  <div>
    <div>
      <h1>
        Navbar
      </h1>
    </div>
  </div>
</body>
`;
